internal/app/dispatcher/run.go

```diff
@@ -41,31 +40,56 @@ func Run(cfg *types.Configuration) {
 
 	var wg sync.WaitGroup
 
-	wg.Add(3)
+	wg.Add(4)
 	go func() {
 		defer wg.Done()
 		for {
+			// Locks are held for 1 mins, renew every 20 sec to keep locks
+			time.Sleep(20 * time.Second)
```

Comment:
Should we provide cancellation contexts to our Sleeps to keep the dispatcher responsive to Kubernetes ops?

Reply:
This is a good point - probably something worthy of a separate issue as quite a chunk of changes unrelated to this PR. It's already tracked https://github.com/lawrencegripper/ion/issues/110 as help wanted.